src/transport_ofi.c

```diff
@@ -292,13 +292,6 @@ static inline int allocate_endpoints(struct fabric_info *info)
 
     /* this endpoint is used only to get read and write
      * counter updates */
-    info->p_info->caps = FI_RMA | FI_WRITE | FI_READ | /*SEND ONLY */
-                         FI_ATOMICS; /* request atomics capability */
-    info->p_info->caps |= FI_REMOTE_WRITE | FI_REMOTE_READ;
-    info->p_info->tx_attr->op_flags = FI_DELIVERY_COMPLETE | FI_INJECT_COMPLETE;
```

Comment:
IIUC, setting the RMA caps is redundant, because we request the following in ```fi_getinfo```: ```FI_RMA | FI_ATOMICS | FI_RMA_EVENT```.  Or is the intent that maybe the domain has additional caps we don't need, so we want to disable them?  Do we need to set ```FI_REMOTE_WRITE | FI_REMOTE_READ``` or does requesting ```FI_RMA_EVENT``` already have this covered?